### PR TITLE
Problem: (CRO-501) Integration tests is not running in HW SGX

### PR DESCRIPTION
**Pull request by calvinlauco**
Solution: Add integration tests in Drone CI running in HW SGX

---
- Integration tests will run as SW mode in TravisCI and HW mode in Drone

### CODE REVIEW

**Comment by calvinlauco** [comment]
bors try

**Comment by calvinaco** [comment]
bors try

**Comment by calvinlauco** [comment]
Enclave is not building correctly in Travis, I will fix it first.

**Comment by calvinaco** [comment]
bors try

File: .drone.yml [hunk 1]
```diff
@@ -18,7 +18,30 @@ steps:
...
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - mkdir bin
+  - export PATH="$PATH:$PWD/bin"
+  - curl -sSL "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o bin/docker-compose
```
Comment (tomtau) [inline .drone.yml]: there's no docker compose installed?

Comment (calvinlauco) [reply .drone.yml]: No, it doesn’t. At least not available for drone executing user. We could install it and save time downloading it every time tests run.

**Pull request merged**